packages/lesswrong/components/common/Header.tsx

```diff
@@ -75,6 +74,19 @@ const styles = (theme: ThemeType): JssStyles => ({
     alignItems: 'center',
     fontWeight: isEAForum ? 400 : undefined,
   },
+  currentEventLink: {
+    marginLeft: "1ch",
+    background: `linear-gradient(91deg,
+      ${theme.palette.text.currentEventHeader.start} 5.84%,
+      ${theme.palette.text.currentEventHeader.stop} 99.75%)
+    `,
+    backgroundClip: "text",
+    "-webkit-background-clip": "text",
+    "-webkit-text-fill-color": "transparent",
```

Comment:
Ew, not CSS's best moment. (I trust you that this is the easiest way to do this, and am upset at CSS.)